core/test/gather/gatherers/inspector-issues-test.js

```diff
@@ -127,7 +127,7 @@ function mockCSP(details) {
 }
 
 /**
- * @param {LH.Crdp.Audits.DeprecationIssueType} type
+ * @param {LH.Crdp.Audits.DeprecationIssueDetails['type']} type
```

Comment:
What happened to this type in the protocol? Why is it only a string now?

Reply:
Yeah I guess it is just a string now